### PR TITLE
Prepares 3.2.0

### PR DESCRIPTION
**Pull request by jdebacker**
Thanks to @MattHJensen @andersonfrailey @codykallen @amshoulders @chusloj @MaxGhenis for your contributions that are added in this release!

### CODE REVIEW

**Comment by jdebacker** [comment]
@MattHJensen I've gone through most of the steps of `RELEASING.MD` (those that lead up to merging this branch).  All unit tests and validation tests pass.  However, I was unable to run `make tctest-jit` without error.  

Also, the document says to run the following:
```
--> cd taxcalc/validation ; ./tests.sh ; .fix. ; cd ../..
```

The `tests.sh` ran without error.  But when I type `.fix.` in my terminal, no command is recognized.  Not sure if I need this or what I'm doing wrong.

**Comment by MattHJensen** [comment]
@jdebacker, the `.fix.` is not a command to the machine but to the human. It's just a note telling you to fix anything that went wrong in the validation. If validation ran without error, then all is good!

**Comment by jdebacker** [comment]
@MattHJensen Last commit makes what I think are the final changes in this PR, updating `RELEASING.MD` to note the steps to releasing a new package on conda-forge.

**Comment by MattHJensen** [comment]
Looks great to me. Thanks for handling this release @jdebacker!

File: docs/contributing/RELEASING.md [hunk 1]
```diff
@@ -30,7 +33,8 @@ RELEASING TAX-CALCULATOR CONDA PACKAGES
 
 --> create release X.Y.Z on GitHub using master branch
 
---> run `pbrelease Tax-Calculator taxcalc X.Y.Z` [to build and upload packages]
+--> Create new package on Conda-Forge for release X.Y.Z
+    --> open a PR to github.com/conda-forge/taxcalc-feedstock where you change the `recipe/meta.yaml` file by updating (1) the version number to X.Y.Z and (2) the checksum to reflectr the checksum for the tarball for release X.Y.Z in the Tax-Calculator GitHub repo
```
Comment (MattHJensen) [inline docs/contributing/RELEASING.md:37]: minor typo reflectr -> reflect

Comment (jdebacker) [reply docs/contributing/RELEASING.md]: Thanks for the catch -- just fixed this.

**Comment by MattHJensen** [comment]
Thanks @jdebacker @andersonfrailey @codykallen @amshoulders @chusloj @MaxGhenis! Merging.

**Pull request merged**